core/src/main/web/outputdisplay/bko-plot/plottip.js

```diff
@@ -96,6 +96,17 @@
         .attr("y1", y1);
     };
 
+
+    var outsideGrid = function (scope, x, y, w, h) {
+      var xPadding = 10;
+      var bBox = scope.jqgridg.get(0).getBBox();
+      var W = bBox.width;
+      var H = bBox.height;
+      var X = bBox.x;
+      var Y = bBox.y;
+      return x > W + X - xPadding || x + w - X + xPadding < 0 || y > H + Y || y + h - Y < 0;
```

Comment:
Could maybe use a comment to explain what are we returning since it's tricky to read this line.